packages/graph/hash_graph/Makefile.toml

```diff
@@ -10,7 +10,7 @@ CARGO_MAKE_CARGO_PROFILE = "production"
 
 [tasks.test]
 run_task = [
-    { name = ["yarn", "deployment-up", "recreate-db", "migrate-up", "test-task", "test-integration", "deployment-down"], condition = { env_true = ["CARGO_MAKE_CI" ] } },
+    { name = ["test-task", "yarn", "deployment-up", "recreate-db", "migrate-up", "test-integration", "deployment-down"], condition = { env_false = ["CARGO_MAKE_CI" ] } },
```

Comment:
Is it possible to run the `test-task` in parallel to starting the DB and such?

Reply:
I think it should be but I worry, that the output will then be messed up.

Reply:
Makes sense, we can hold off for now then